<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>699</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="txtTitulo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>488</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Sans</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Simulador de Planificación de Procesos</string>
    </property>
   </widget>
   <widget class="QLabel" name="textNumProcesos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>143</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Número de Procesos:</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderNumProcesos">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="numProcesos">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSimular">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>105</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(21, 109, 12);
color: rgb(255,255,255);
</string>
    </property>
    <property name="text">
     <string>SImular</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>1281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>1281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="AlgorithmSelect">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>363</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Primero en llegar primero en ejecutar (FCFS)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Primero el trabajo más corto (SJF)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Selección aleatoria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Planificación basada en prioridades (No Expulsivo)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Turno rotativo (round robin)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Primero el menor tiempo restante (SRTF)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Planificación basada en prioridades (Expulsivo)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="txtAlgoritmo">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>72</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Algoritmo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtResultados">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resultados</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtEspera">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tiempo de espera:  </string>
    </property>
   </widget>
   <widget class="QLabel" name="txtEsperaAvg">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>540</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Promedio:  0ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtEsperaMin">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mínimo:  0ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtEsperaMax">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>540</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Máximo:  0ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtRespuesta">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>156</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tiempo de Respuesta:  </string>
    </property>
   </widget>
   <widget class="QLabel" name="txtRespuestaMax">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>570</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Máximo:  0ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtRespuestaAvg">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>570</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Promedio:  0ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtRespuestaMin">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>570</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mínimo:  0ms</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>1281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>1281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="txtTiempoTotal">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tiempo Total:  0ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtInactivo">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>630</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inactivo: 0ms [0.0%]</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtEjecutando">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>630</y>
      <width>301</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ejecutando CPU: 0ms [0.0%]</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>1281</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
